src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	
+	private String blkNum;
+	
+	public Block(String blockNumber){
+		this.blkNum = blockNumber;
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.